workers/data_refinery_workers/processors/smasher.py

```diff
@@ -485,22 +493,25 @@ def _smash(job_context: Dict, how="inner") -> Dict:
 
         # Smash all of the sample sets
         logger.debug("About to smash!",
-                     input_files=job_context['input_files'],
-                     dataset_data=job_context['dataset'].data,
+                     input_files=job_context['input_files'].keys(),
+                     dataset_data=len(job_context['dataset'].data),
         )
 
-        job_context['technologies'] = {'microarray': [], 'rnaseq': []}
+        #job_context['technologies'] = {'microarray': [], 'rnaseq': []}
```

Comment:
?

Reply:
Probably just meant to remove this right?

Reply:
Or just leave it uncommented so we can see that it's going to be in the context like that.